packages/contracts-bedrock/test/fixtures/L2GenesisDeployedBytecode.sol

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
This is not going to be very scalable because every time we change the source code, we will need to modify these constants